docs/designs/link-output.md

```diff
@@ -42,11 +42,14 @@ Docfx generate `links` information of each files into one single file `.links.js
 
 *link model*:
 
-  |                         |                      |
-  |-------------------------|----------------------|
-  | `source_url`            | /dotnet/articles/a   |
-  | `source_moniker_group`  | 218c13d0             |
-  | `target_url`            | /dotnet/articles/b   |
+  |                         |                                         |
+  |-------------------------|----------------------                   |
+  | `source_url`            | /dotnet/articles/a                      |
+  | `source_moniker_group`  | 218c13d0                                |
+  | `target_url`            | /dotnet/articles/b                      |
+  | `source_file_url`       | https://github.com/dotnet/articles/a.md |
+  | `source_line`           | 10                                      |
```

Comment:
Say `a.md` includes `b.md`. Will `source_line` be the line of `[!include]` in `a.md`, or the line of actual broken link in `b.md`?

Reply:
the source file url should point to b.md and source line should be the line in b.md, that's my understanding.

Reply:
> the source file url should point to b.md and source line should be the line in b.md, that's my understanding.

agree